packages/atlas/src/components/_nft/NftTileViewer/NftTileViewer.tsx

```diff
@@ -39,74 +48,57 @@ export const NftTileViewer: React.FC<NftTileViewerProps> = ({ nftId }) => {
   }
 
-  const { currentBlock } = useJoystream()
-  const { convertBlockToMsTimestamp } = useBlockTimeEstimation()
-  const msTimestamp = useMsTimestamp()
-
-  const getNftProps = (): NftTileProps => {
-    const nftCommonProps = {
-      ...nftStatus,
-      loading: loading || !nftId,
-      thumbnail: {
-        videoHref: absoluteRoutes.viewer.video(nft?.video.id),
-        thumbnailUrl: thumbnail.url,
-        loading: thumbnail.isLoadingAsset,
-        thumbnailAlt: `${nft?.video?.title} video thumbnail`,
-      },
-      owner: nft?.ownerMember?.id
-        ? {
-            name: nft?.ownerMember?.handle,
-            assetUrl:
-              nft?.ownerMember?.metadata.avatar?.__typename === 'AvatarUri'
-                ? nft.ownerMember?.metadata.avatar.avatarUri
-                : '',
-            loading,
-            onClick: () => navigate(absoluteRoutes.viewer.member(nft?.ownerMember?.handle)),
-          }
-        : undefined,
-      creator: {
-        name: nft?.video.channel.title,
-        loading: creatorAvatar.isLoadingAsset || loading,
-        assetUrl: creatorAvatar.url,
-        onClick: () => navigate(absoluteRoutes.viewer.channel(nft?.video.channel.id)),
-      },
-    } as NftTileProps
-    switch (nft?.transactionalStatus.__typename) {
-      case 'TransactionalStatusIdle':
-      case 'TransactionalStatusInitiatedOfferToMember':
-      case 'TransactionalStatusBuyNow':
-        return {
-          ...nftCommonProps,
-        }
-      case 'TransactionalStatusAuction': {
-        const auctionPlannedEndBlock = nftStatus?.status === 'auction' && nftStatus?.auctionPlannedEndBlock
-        const isEnded = auctionPlannedEndBlock && currentBlock >= auctionPlannedEndBlock
-        const plannedEndMsTimestamp =
-          !isEnded && !!auctionPlannedEndBlock && convertBlockToMsTimestamp(auctionPlannedEndBlock)
-
-        return {
-          ...nftCommonProps,
-          status: isEnded ? 'idle' : 'auction',
-          buyNowPrice: isEnded ? undefined : Number(nft.transactionalStatus.auction?.buyNowPrice),
-          startingPrice: Number(nft.transactionalStatus.auction?.startingPrice),
-          topBid: Number(nft.transactionalStatus.auction?.lastBid?.amount),
-          timeLeftMs: plannedEndMsTimestamp ? plannedEndMsTimestamp - msTimestamp : undefined,
-        }
-      }
-      default:
-        return {
-          ...nftCommonProps,
+  const nftCommonProps: NftTileProps = {
+    ...nftStatus,
+    loading: loading || !nftId,
+    thumbnail: {
+      videoHref: absoluteRoutes.viewer.video(nft?.video.id),
+      thumbnailUrl: thumbnail.url,
+      loading: thumbnail.isLoadingAsset,
+      thumbnailAlt: `${nft?.video?.title} video thumbnail`,
+    },
+    owner: nft?.ownerMember?.id
+      ? {
+          name: nft?.ownerMember?.handle,
+          assetUrl:
+            nft?.ownerMember?.metadata.avatar?.__typename === 'AvatarUri'
+              ? nft.ownerMember?.metadata.avatar.avatarUri
+              : '',
```

Comment:
I know this is a carry-over from the previous code but avatar from storage system should be handled here as well. Can you add a TODO comment somewhere so this doesn't get lost?